.travis.yml

```diff
@@ -2,7 +2,6 @@ language: scala
 
 scala:
 - 2.11.7
-- 2.12.0-M2
```

Comment:
Some tests didn't pass anymore in `SerializationTests`, I just pushed some workarounds (and reverted this particular change).